src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery extends ColorSupplier {
```

Comment:
Random and ColorSupplier should be class fields
```suggestion
public class Lottery {
```

Reply:
done